validate.js

```diff
@@ -0,0 +1,40 @@
+const form = document.getElementById('form');
+const email = document.getElementById('contact-email');
+const userName = document.getElementById('contact-name');
+const message = document.getElementById('form-message');
+const error = document.getElementById('error-message');
+
+error.style.visibility = 'hidden';
+
+function formValidate(e) {
+  if (email.value !== email.value.toLowerCase()) {
+    error.style.visibility = 'visible';
+    error.innerHTML = 'Please enter your email address in Lowercase.';
+    e.preventDefault();
+  } else {
+    error.style.visibility = 'hidden';
+  }
+}
+
+form.addEventListener('submit', formValidate);
+
+function storeInput() {
+  const user = {
+    UserEmail: email.value,
+    UserName: userName.value,
+    UserText: message.value,
+  };
+  localStorage.setItem('user', JSON.stringify(user));
```

Comment:
- [ ] It would be better if we can see our object saved in our local storage. However, I'm unable to find the object saved in the browser localstorage as shown in the screenshot below
![Screenshot from 2022-09-17 18-31-57](https://user-images.githubusercontent.com/46388113/190871736-9484a7f1-49ab-45a3-be0a-3b224045892d.png)